packages/@react-spectrum/s2/chromatic/CheckboxGroup.stories.tsx

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Checkbox, CheckboxGroup, Content, ContextualHelp, Heading} from '../src';
+import {generateComboChunks, shortName} from './utils';
+import type {Meta} from '@storybook/react';
+import {style} from '../style/spectrum-theme' with { type: 'macro' };
+
+const meta: Meta<typeof CheckboxGroup> = {
+  component: CheckboxGroup,
+  parameters: {
+    chromaticProvider: {disableAnimations: true}
+  },
+  title: 'S2 Chromatic/CheckboxGroup'
+};
+
+export default meta;
+
+let states = [
+  {isDisabled: true},
+  {isEmphasized: true},
+  {isInvalid: true},
+  {isReadOnly: true},
+  {isRequired: true},
+  // {labelAlign: ['start', 'end']},
+  // {labelPosition: ['top', 'side']},
```

Comment:
commented this out for now since it proved difficult to implement with the powerset given the sizing and/or label size. Probably should have a subset of dedicated stories but to be done later